Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -15,10 +15,7 @@ networking is setup, the Cilium CNI plugin is called to attach eBPF programs to
 the network devices set up by aws-cni to enforce network policies, perform
 load-balancing, and encryption.
 
-.. note::
-
-   When running Cilium in chaining configuration on top of AWS VPC CNI, the L7 policies may not work. 
-   This limitation is currently tracked at `#12454 <https://github.com/cilium/cilium/issues/12454>`_.
+.. include:: cni-chaining-limitations.rst
```

Comment:
Do we need the same in `Documentation/gettingstarted/cni-chaining-portmap.rst` as well?

Reply:
My understanding is that portmap is handled differently from the other chaining plugins, as it only handles some service bits to allow the host port mapping to occur; usually Cilium is still responsible for networking in that case so it can successfully provision everything correctly for the L7 and/or encryption to work.